src/frontend/game/GameApp.tsx

```diff
@@ -1,14 +1,18 @@
 import { type FunctionalComponent } from 'preact';
 import { ClientContextProvider } from 'src/frontend/game/ClientContextProvider.tsx';
-import { PlayerList } from 'src/frontend/game/components/PlayerList';
```

Comment:
if you're using this pr to learn how to create a new game event this file can be ignored, the changes are unrelated to implementing the ChangePlayerNameEvent.